chili-epfl/FROG
# More validation in the createAccount function on the server

This PR adds more validation to the server side create account method. This prevents potential issues when the user tries to create an account with invalid parameters using the console as @ajainuary pointed out. Also `validationHelpers.js ` has been moved to `frog-utils`. This file contains a lot of useful functions to validate emails, password, display name.  Also did a little bit of clean up to the `formValid` function in `SignUp.js`. 

**Testing instructions**
- Use `Meteor.call('create.account', 'someinvalidemail','somepassword',{someRandomObject:'Hello'})` on the console. If the parameters are invalid, an error message will be returned. 
- Try to create an account with no display name/ no password and an error message should be returned.
